src/CommunityPatchLauncher/Windows/MainWindow.xaml.cs

```diff
@@ -78,12 +79,11 @@ private void DownloadFileAsync(string URI, string File, string Name, bool Settle
 
         private void DownloadFileEventCompletedUnited(object sender, AsyncCompletedEventArgs e)
         {
-            //ToDo Install first HE!!! @Leonards05 | Pumpline#5578
-            
+            var filename = System.Environment.GetEnvironmentVariable("TEMP") + "\\SettlersUnitedSetup.exe";
```

Comment:
You should use Path.GetTempPath() to get the temp path, it's more reliable
You should create system file paths with Path.Combine()